src/game/engine/techno.h

```diff
@@ -52,6 +52,16 @@ enum VisualType
     VISUAL_HIDDEN,
 };
 
+class KillTrackerClass
```

Comment:
Move this to its own file, it can be a header only class given that it is all inline. Personally I think this level of encapsulation is over the top and it was fine as a single variable unless you wanted to start making Register_Kill do other stuff. Its also entirely possible this wasn't actually present in RA and was added after the code bases diverged between this and sole.